geopandas/base.py

```diff
@@ -2269,6 +2269,111 @@ def distance(self, other, align=True):
         """
         return _binary_op("distance", self, other, align)
 
+    def frechet_distance(self, other, align=True, densify=None):
+        """Returns a ``Series`` containing the Frechet distance to aligned `other`.
+
+        The Frechet distance is the largest distance consisting of any point in `self`
+        with the nearest point in `other`.
+        The operation works on a 1-to-1 row-wise manner:
+        .. image:: ../../../_static/binary_op-01.svg
+           :align: center
+        Parameters
```

Comment:
This docstring is a bit broken. I think you somehow lost empty lines between the sections. This will not render properly.

Reply:
thanks for pointing it out, added back the empty lines :)